tutorial/index.md

```diff
@@ -21,10 +21,10 @@ unstuck!
 ## Instructors
 
 The instructors for this workshop are
-- Jordão Bragantini
-- Erick Martins Ratamero
 - Stefan van der Walt
```

Comment:
Swap Lars and Stéfan, so Lars appears first.

Reply:
I thought to list them in speaking order. But I don't mind switching them around.